core/setup.py

```diff
@@ -43,9 +43,7 @@
     include_package_data=True,
     test_suite="test",
     entry_points={
-        "console_scripts": [
-            "dbt = dbt.main:main",
-        ],
+        "console_scripts": ["dbt = dbt.main:main", "dbt-cli = dbt.cli.main:cli"],
```

Comment:
I'm not sure if it's kosher to include a second  cli entry point given it's not complete yet-- @jtcohen6 do you have feelings on this?

Reply:
jc- how would `dbt-cli` get called?

Reply:
You would type it on the command line.  Each entry to `console_scripts` gets registered as a thing-you-can-run-from-the-command-line`.
https://setuptools.pypa.io/en/latest/userguide/entry_point.html

Reply:
What would the alternative be, in order to test out the new CLI? Is there a way to invoke it via module (`python3 -m dbt.cli.main`) instead?

- If we're planning to merge this before v1.3.0-rc1, I'd prefer _not_ to add a new incomplete CLI entry point
- If we're waiting to merge this until after we've cut v1.3.0-rc1, I think it's fine to have it in `main` (alpha state) while we work on completing the new CLI

Reply:
> Is there a way to invoke it via module (python3 -m dbt.cli.main) instead?
It's not set up as a runnable module right now, but that's just a two-line edit.

Why don't I go ahead and make it runnable as a module and remove the direct entrypoint for now since I'm pretty sure we're merging this pre-1.3.0-rc1.

Reply:
Updated.